wordpress-mobile/WordPress-iOS
# Unit Tests for SuggestionsTableView

## Issue
Part of #18979

## Description
This adds Unit Tests to `SuggestionsTableView` most critical methods, which are:
- showSuggestions(forWord:)
- searchResults(searchQuery:suggestions:suggestionType)
- moveProminentSuggestionsToTop(searchResults:prominentSuggestionsIds:)
- prominentSuggestions(fromPostAuthorId:commentAuthorId:defaultAccountId:)

## Test Instructions
- Make sure `WordPress` target is selected
- Show the Test Navigator
- Filter by `SuggestionsTableViewTests`
- Run Tests
- Verify that all tests are green ✅ 

## Regression Notes
1. Potential unintended areas of impact
n/a

3. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

4. What automated tests I added (or what prevented me from doing so)
n/a

## PR submission checklist:
